### DISCUSSION TITLE
Python support

### DISCUSSION THREAD

**Discussion by sapal6** [cypress-io/cypress#16320] [General]
Hi,
I have recently started to explore Cypress and I am quite liking it. Although I am comfortable with coding tests in Typescript but I would like to know if there is any plan to support Python or if there is any ongoing work in this direction ?

By the way thank you for creating this beautiful tool for the test community.

**Comment by lmiller1990** [comment]
Hi. Glad you are enjoying Cypress.

To my knowledge, there is no plans for python bindings on the roadmap at this point in time.
